stable/mongodb-replicaset/values.yaml

```diff
@@ -49,14 +49,31 @@ persistentVolume:
+  cakey: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcGdJQkFBS0NBUUVBNHU0WXhhQTNpd0ExcnVPU3BOM1ZTdXQ4NnJQWlpDbjc0Q3ppTlRPc0xTMmFGY1dOCmhTbHY4UzNFK2pwdGkzUW4rY3RhYUd6UEVsRGhYamN1M01BNDNmQUZIK1ZJVmYvNTBMRG9JRVdFN1IrclV2SEoKTnB1ZEdTU2JmdUZKNzIrMTFZT3c0UktKYU1ETnNLeEdZU3VVTVB2eWJodUxlSmVBZnBZVUE2M3Z2NEpDRE5iMQpVRWlUUmhWaVMreE5XeDk1Y1U4WVp4YkpCSk50ME1XRG9xcTdIQzBVRXhRb0xVNG1MUjE5b3BISk5aaXQ2OGRUCkE3akNnb3pzYkhieXNKN09lVFBzWjQ5VXNnT3VSVkZQd0R6OFlrSnJLc21LVmdXNlVQNTMxSHFiK08yUG42aWYKTWJzOUpNMXp4aFhWMUUxLzJxZE5oYTdiN08xSzJJem9xbkZZR3dJREFRQUJBb0lCQVFDWlNsaTFkcXJPWEkvbQoyWmMvZ3hJSGo5ZEwrNzFIcWd5WXUvajZFek9sT1A2eTRrWGN3OGI0VmdDUFAxU3NOUVNqK2pmUi9sNUtjeFFwCnZvRUxZVldJcGcvd0ljT2hSRTZPS0ZKRktuREdkUFFZRFhmc1NXWHZqQWoxeGwxWlBrN0lNak1UZHFBWGdOWHgKR0E1YXFVTnJhMld4OXkxWmdvMnI5TmtFTzBTT1Nic0UvaTZjVmI4NWE1TW5OS2swbDRDQjJ2Sm5rNUhkTjR3RgpLa3F3ZGZxOW91dER2U0pydWZ3aWpWNGxicktjWHFiUFhweFV1MUh4RnBHaGV5MThYUmtKSEc0Nnljc3k0bVM1CmM1Z2ZCYkZOTk5MZENhNThzWHNDaUdjWW1SNk5jVDFIZ05YdmxSK1JrUXBzbWg4SU5idUltemZKK0ZkN01TKy8KTjZmUGh4YWhBb0dCQVA5S1NXK2lGRzhybEdqYWN1VXIweFhhWGc2bnVSZGhmSG1wNkMxUDY4dDFoa2NZZGNGSApSTFErdVFFa0diWUJFYzZjbnk5U0tvM1U4Sm1UdDM2UU96eklyNk9QQmRpMFVHUzJiK09kYWs1MndPYUhJSzNSCnlKL2g0eFVIRnRLckMrS3o4SXJuQ3B2dEZKc0ZQMG1td3JkaHpEY2hBakdMcGtTSzRGRkZFSHJaQW9HQkFPT1AKbjVubnhpNElEV2pmRHJQSHpxSFRvOUIrN0h2RndTcUJTdWlaRlN3ZCtVU3dSaXNyZnczcEhDbnllZ1NMc0J1UQpsZWlIWU9aMWcvbHNPWi8yUnFYRURjUW1KdnRoUWZMZ1cxN1NHUHZJMS8zKzYwTFZXOG00RTVIa0M0OFJ1RFdZCnA5NHB6YXpsREpHU0FiTStiMHNycitsY3dGMVZndlp1OEpFYjhjb1RBb0dCQUt1eUhiUEwwV3gzWmVnalMzVGcKUmFhRDZCQjVNMTRMUGJsaDBYcHdMd2oyc09tdUJMeUtIeVlPclVTY01BS2RzNGsvQXp6RFNVVjh1cGkvcFlDawpwb0thQXE3QktWQjI3VDdmQ0YrUVpvZ0JESG0rMWZheGlmb2w1U0VtVkc0NWhkUGg2MXl3U1ExWHBTMGhtVkFDCnJTREMwS01mT3lkeFlkRGxpanArZGJmNUFvR0JBSVh2YWFPZkxDQzVzcExQL29wSUJ0RnVqSk5Sc1FTa2Z5M3kKVFZHYmtEK1lJTGVSL0txNS9oMEU0QU84YUo2MlROL2NlSzhVeHk2OWFnRHE5dnUwYXg1enh4cDB6aXJVc29rdQpEZmxBTlY0MGR4UlM3c3MyWXNEV2JBWkpMNWgwZUhSck5ocG5obHZGc0dFWmk1Q2hKY1libjN6OWoyY2Z4bVlmCkk4NTBvUm9yQW9HQkFNLzhxUmw4SFRIY0ZIbFplZ0F1MXBjN29zR0hwQ3ZYTDlWbHZJVTBSRVpDUjJFMHI0SkQKWDd3bjB0SHFsYUZkd1RyWXNWUWIvMUlVZWpqeDNJOCtVcEFoanBINmRyMGZtaHZqazlYQ1hzVjlPQTNIYmNOOQpOczBsaDVQL3QvTU1PU3E1UEkvOTZXVXo5L1dtZlk5anNDc1laQVJIamxPNk96MkQ5YlYzNCswNAotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
+
 # Entries for the MongoDB config file
 configmap:
   storage:
     dbPath: /data/db
   net:
     port: 27017
+    ssl:
+    # Uncomment for TLS support
+    # mode: requireSSL
+    # CAFile: /ca/tls.crt
+    # PEMKeyFile: /work-dir/mongo.pem
   replication:
     replSetName: rs0
-# security:
-#   authorization: enabled
-#   keyFile: /keydir/key.txt
+  security:
+  # Uncomment for TLS support
```

Comment:
The comment is a little bit misleading because this is also needed for key file access control.